config.toml.example

```diff
@@ -1,6 +1,6 @@
 [app]
 # Relay listen address
-#address = "127.0.0.1:1337"
+#addresses = ["127.0.0.1:1337", "[::1]:1337"]
```

Comment:
Please use localhost for the default and add a comment saying that multiple addresses may be specified.